packages/mdc-menu-surface/index.js

```diff
@@ -258,10 +258,10 @@ class MDCMenuSurface extends MDCComponent {
         return {x: window.pageXOffset, y: window.pageYOffset};
       },
       setPosition: (position) => {
-        this.root_.style.left = 'left' in position ? position.left : null;
-        this.root_.style.right = 'right' in position ? position.right : null;
-        this.root_.style.top = 'top' in position ? position.top : null;
-        this.root_.style.bottom = 'bottom' in position ? position.bottom : null;
+        this.root_.style.left = 'left' in position ? `${position.left}px` : '';
```

Comment:
Why did this change from `null` to `''` here, if we're not doing that in the TypeScript PR? (It occurred to me to suggest this there, but I figured we were preserving existing behavior.)

Reply:
The Closure Compiler test suddenly started complaining that these values can't be `null` 🤷‍♂️

Reply:
CSS specs seem to indicate that `null` is _not_ a valid value. Empty string is preferred.
So Closure Compiler is [technically correct](https://i.imgur.com/PpV0evc.png); [TypeScript is wrong](https://github.com/Microsoft/TypeScript/blob/2f9218f346bbc606cccc4aebfca1c2c4f3167fee/lib/lib.dom.d.ts#L2435-L2436).

* https://developer.mozilla.org/en-US/docs/Web/API/CSSStyleDeclaration/setProperty

    > `value` _Optional_ is a `DOMString` containing the new property value. If not specified, treated as the **empty string**.

* https://developer.mozilla.org/en-US/docs/Web/API/CSSStyleDeclaration/getPropertyValue

    > Return value is a `DOMString` containing the value of the property. If not set, returns the **empty string**.

* https://drafts.csswg.org/cssom/#the-cssstyledeclaration-interface
* https://drafts.csswg.org/cssom/#cssomstring
* https://heycam.github.io/webidl/#idl-DOMString

    > Note: Note also that `null` is not a value of type `DOMString`. To allow `null`, a nullable `DOMString`, written as `DOMString?` in IDL, needs to be used.